tests/documentation/test_files/DocumentationStyleBear/bad_file3.py.test

```diff
@@ -0,0 +1,19 @@
+def b():
+    """
+    :param t:xyz
+    :return:123"""
+
+
+class Doc():
+    """
+    Example Class
+    :param t:xyx
+    :return:215
+    """
+
+    def D1():
+        """
+        This is nested function
```

Comment:
no it isnt.  It is an error.  It needs to either be a staticmethod or have a self parameter.